docs/build/nodes/evm-tracing-node.md

```diff
@@ -11,6 +11,11 @@ import TabItem from '@theme/TabItem';
 
 Running a tracing node on an Astar chain allows you to debug EVM transactions and have enhanced access to transaction pool using [EVM debug RPC](/docs/build/EVM/evm-debug-api).
 
+:::warning
```

Comment:
@Dinonard can you make sure this is displayed correctly in rich mode

Reply:
Not sure what you mean?

Docusaurus displays is correctly I think?
<img width="1838" alt="Screenshot 2025-02-04 at 11 32 26" src="https://github.com/user-attachments/assets/ec412e1c-d3d2-4b26-ba93-d09cdc970a8b" />

Reply:
Nevermind, I was trying rich mode